encoder-otel-brave/src/main/java/zipkin2/reporter/otel/brave/SpanTranslator.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.reporter.otel.brave;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import brave.Span.Kind;
+import brave.Tag;
+import brave.handler.MutableSpan;
+import brave.handler.MutableSpan.AnnotationConsumer;
+import brave.handler.MutableSpan.TagConsumer;
+import brave.http.HttpTags;
+import com.google.protobuf.ByteString;
+import io.opentelemetry.proto.common.v1.AnyValue;
+import io.opentelemetry.proto.common.v1.InstrumentationScope;
+import io.opentelemetry.proto.common.v1.KeyValue;
+import io.opentelemetry.proto.trace.v1.ResourceSpans;
+import io.opentelemetry.proto.trace.v1.ResourceSpans.Builder;
+import io.opentelemetry.proto.trace.v1.ScopeSpans;
+import io.opentelemetry.proto.trace.v1.Span;
+import io.opentelemetry.proto.trace.v1.Span.Event;
+import io.opentelemetry.proto.trace.v1.Span.SpanKind;
+import io.opentelemetry.proto.trace.v1.Status;
+import io.opentelemetry.proto.trace.v1.TracesData;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.semconv.HttpAttributes;
+import io.opentelemetry.semconv.NetworkAttributes;
+import io.opentelemetry.semconv.ServerAttributes;
+import io.opentelemetry.semconv.ServiceAttributes;
+import io.opentelemetry.semconv.UrlAttributes;
+
+
+/**
+ * SpanTranslator converts a Brave Span to a OpenTelemetry Span.
+ */
+final class SpanTranslator {
+
+  // Defined in the incubating SDK https://github.com/open-telemetry/semantic-conventions-java/blob/main/semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/PeerIncubatingAttributes.java
+  static final String PEER_SERVICE = "peer.service";
+
+  private static final Map<String, String> RENAMED_LABELS;
+
+  static {
+    RENAMED_LABELS = new LinkedHashMap<>();
+    // "http.host" is not defined in HttpTags, but is a well-known tag.
```

Comment:
I assumed that `http.tag` was used in Brave's code because it's commonly used in test data, but now I found that it's only used in other languages.
https://github.com/search?q=org%3Aopenzipkin+%22http.host%22&type=code

Maybe we can remove this conversion?